src/d3a/d3a_core/area_serializer.py

```diff
@@ -141,7 +143,9 @@ def optional(attr):
         area = Area(name, children, uuid, optional('strategy'), optional('appliance'), config,
                     optional('budget_keeper'), grid_fee_percentage=grid_fee_percentage,
                     transfer_fee_const=grid_fee_constant,
-                    external_connection_available=external_connection_available)
+                    external_connection_available=external_connection_available,
+                    baseline_peak_energy_import_kWh=baseline_peak_energy_import_kWh,
```

Comment:
I assume that due to the GraphQL issue we discussed yesterday, you decided to have individual constructor arguments here too, right ?

Reply:
`due to the GraphQL issue we discussed yesterday, you decided to have individual constructor arguments in the constructor of the Area class too, right `  Exactly. I could think of an object that can hold several parameters that is not a JSON string. 
I had it all implemented here, but reversed it due to our discussion.